### PR TITLE
feat: xcm pahse 1 for sibling chain

### PR DESCRIPTION
**Pull request by wangminqi**

### CODE REVIEW

**Comment by wangminqi** [comment]
This update should enable the ability of single side xcm (cross LIT asset to other chain)

**Review by Kailai-Wang — APPROVED** [review]
Approved.
This will also be addressed in the `polkadot-v0.9.18` update later.

**Review by Kailai-Wang — CHANGES_REQUESTED** [review]
Oh sec, you are actually mixing up runtime:
`lib.rs` for litentry runtime and `xcm_config` for litmus were modified.

You probably want to change both

File: runtime/litmus/src/xcm_config.rs [hunk 1]
```diff
@@ -149,8 +150,10 @@ impl pallet_xcm::Config for Runtime {
 	// ^ Disable dispatchable execute on the XCM pallet.
 	// Needs to be `Everything` for local testing.
 	type XcmExecutor = XcmExecutor<XcmConfig>;
-	type XcmTeleportFilter = Everything;
-	type XcmReserveTransferFilter = Nothing;
+	type XcmTeleportFilter = Nothing;
+	// This filter here defines what is allowed for XcmExecutor to handle with TransferReserveAsset
+	// Rule.
+	type XcmReserveTransferFilter = Everything;
```
Comment (suinuj) [inline runtime/litmus/src/xcm_config.rs:156]: Just a question here, so the code change is to enable the reserve transfer and disable teleport, right?

Comment (wangminqi) [reply runtime/litmus/src/xcm_config.rs:156]: Yes. Teleport  means burn/mint for XCM token bridge handling native Litmus asset. Reserve means deposit/withdraw using remote chain's litmus sovereign account.

**Comment by wangminqi** [comment]
> Oh sec, you are actually mixing up runtime: `lib.rs` for litentry runtime and `xcm_config` for litmus were modified.
> 
> You probably want to change both

You are right. Updated.

**Review by Kailai-Wang — APPROVED** [review]

**Pull request merged**